templates/ReactReduxSpa/ClientApp/configureStore.ts

```diff
@@ -1,27 +1,29 @@
-import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
-import * as thunkModule from 'redux-thunk';
-import { routerReducer } from 'react-router-redux';
-import * as Store from './store';
-import { typedToPlain } from 'redux-typed';
+import * as Redux from "redux";
+import { createStore, applyMiddleware, compose, combineReducers, GenericStoreEnhancer } from "redux";
+import thunk from "redux-thunk";
+import { routerReducer } from "react-router-redux";
+import * as Store from "./store";
+import { typedToPlain } from "redux-typed";
+
```

Comment:
Haha sounds good, be right back (grabbing lunch) and I'll fix the above comments :+1: